rules/0575-windows_rules.xml

```diff
@@ -0,0 +1,5170 @@
+  </rule>
+
+  <rule id="20399" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1022$</field>
+    <description>Update installed successfully</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20400" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1024$</field>
+    <description>Update could not be installed</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20401" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1025$</field>
+    <description>File in use by another program</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20402" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1026$|^1027$</field>
+    <description>The configuration data registry key, sub key or configuration data cache folder were not secured properly</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20403" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1029$</field>
+    <description>The installation or update for the product required a restart for all changes to take effect</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20404" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1030$</field>
+    <description>The application tried to install a more recent version of the protected Windows file</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20405" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1031$</field>
+    <description>Assembly component is in use</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20406" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1032$</field>
+    <description>Environment variables updated during installation generated an error</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20407" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1033$</field>
+    <description>Windows Installer installed the product</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20408" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1035$</field>
+    <description>Windows installer reconfigured the product</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20409" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1036$</field>
+    <description>Windows installer installed an update</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20410" level="3">
+    <if_sid>20382</if_sid>
+    <field name="win.system.eventID">^1037$</field>
+    <description>Windows installer removed an update</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20411" level="4">
```

Comment:
Level 0 -> I don't think it is necessary to fire a rule to notice the Windows machine has to be restarted to apply changes. It should be logged in the host locally.